.gitignore

```diff
@@ -1,2 +1,7 @@
 .jekyll-cache
 _site
+_data/*/
+!_data/en/
```

Comment:
You may want to exclude the README also.

Reply:
No need. The trailing slash on `_data/*/` only matches directory names to ignore, so plain files within `_data` are not ignored. That's the reason I included the trailing slash.

Reply:
Ok. Fair point.